### ISSUE TITLE
Add note about restarting after changing setting

### ISSUE THREAD

**Issue by montanaflynn** [p0deje/Maccy#27]
```
defaults write org.p0deje.Maccy hotKey command+shift+v
killall Maccy
open /Applications/Maccy.app
```

**Comment by p0deje** [comment]
Huh, this should probably be different so that you don't need to restart at all.

**Comment by montanaflynn** [comment]
That would be even better. 

Thanks for the great piece of software, I've been looking for something like this for a long time!

**Comment by DaFuqtor** [comment]
That's it? https://developer.apple.com/documentation/foundation/userdefaults/1408206-didchangenotification

**Comment by p0deje** [comment]
> That's it? https://developer.apple.com/documentation/foundation/userdefaults/1408206-didchangenotification

I don't think so:

> This notification isn't posted when changes are made outside the current process

**Comment by p0deje** [comment]
Fixed in 01a9091
